specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2022-03-08-preview/FlexibleServers.json

```diff
@@ -0,0 +1,1017 @@
+          "description": "The SKU (pricing tier) of the server."
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "Describes the identity of the application."
+        },
+        "properties": {
+          "$ref": "#/definitions/ServerPropertiesForUpdate",
+          "x-ms-client-flatten": true,
+          "description": "Properties of the server."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Application-specific metadata in the form of key-value pairs."
+        }
+      },
+      "description": "Represents a server to be updated."
+    },
+    "ServerListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Server"
+          },
+          "description": "The list of flexible servers"
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of operations.",
+          "type": "string"
+        }
+      },
+      "description": "A list of servers."
+    },
+    "Sku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the sku, typically, tier + family + cores, e.g. Standard_D4s_v3."
+        },
+        "tier": {
+          "type": "string",
+          "description": "The tier of the particular SKU, e.g. Burstable.",
+          "enum": [
+            "Burstable",
+            "GeneralPurpose",
+            "MemoryOptimized"
+          ],
+          "x-ms-enum": {
+            "name": "SkuTier",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "name",
+        "tier"
+      ],
+      "description": "Sku information related properties of a server."
+    },
+    "Storage": {
+      "type": "object",
+      "properties": {
+        "storageSizeGB": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Max storage allowed for a server."
+        }
+      },
+      "description": "Storage properties of a server"
+    },
+    "AuthConfig": {
+      "type": "object",
+      "properties": {
+        "activeDirectoryAuthEnabled": {
+          "type": "boolean",
+          "description": "If true, Azure Active Directory authentication is enabled."
+        },
+        "passwordAuthEnabled": {
+          "type": "boolean",
+          "default": "",
+          "description": "If true, Password authentication is enabled."
+        },
+        "tenantId": {
+          "type": "string",
+          "default": "",
+          "description": "Tenant id of the server."
+        }
+      },
+      "description": "Authentication configuration properties of a server"
+    },
+    "DataEncryption": {
+      "type": "object",
+      "properties": {
```

Comment:
Are we adding `geoBackupUserAssignedIdentityId` and `geoBackupKeyURI` for this api version?

Reply:
No we are not adding these